pkg/controllermanager/controllermanager.go

```diff
@@ -186,6 +187,7 @@ func NewControllerInitializers() controllercontext.Initializers {
 		initializers["deployer"] = startDeployerController
 	}
 	initializers["clusterdiscovery"] = startClusterDiscoveryController
+	initializers["metricsexpose"] = startMetricsServer
```

Comment:
It should not be here.

Reply:
use NewHealthzAndMetricsHandler instead of metricsServer.